src/Umbraco.Web.UI.Client/examples/entity-content-type-condition/workspace-view-unique.element.ts

```diff
@@ -0,0 +1,20 @@
+import { html, customElement } from '@umbraco-cms/backoffice/external/lit';
+import { UmbLitElement } from '@umbraco-cms/backoffice/lit-element';
+
+@customElement('umb-example-entity-content-type-condition-unique')
+export class UmbWorkspaceExampleViewUniqueElement extends UmbLitElement {
+	override render() {
+		return html`<uui-box>
+			<h3>Content Type Unique Condition Test</h3>
+			<p>It appears only on documents with GUID: <strong>42d7572e-1ba1-458d-a765-95b60040c3ac</strong></p>
```

Comment:
As this example is a bit different compared to the others, the developer has to update the example to make it fit their Content Type uniquely. Then, I think it would make the example better if we also rendered the “Real” Content Type unique in the Workspace View. Then we don't end up in any confusion about what the developer entered in the condition example doesn't match what is shown.

Instead of hardcoding it here, I think you can observe the uniques the same way you do in the condition and then print them here.